### DISCUSSION TITLE
Automation for release forms with query string values - previous tag

### DISCUSSION THREAD

**Discussion by asos-danielc** [community/community#102124] [Repositories]
### Select Topic Area

Question

### Body

I have read the documentation found here https://docs.github.com/en/repositories/releasing-projects-on-github/automation-for-release-forms-with-query-parameters

What would be release nice is to be able to specify a query string value for the previous release to be auto selected from the 'Previous tag' for releases and in turn generating release notes.

Is there an undocumented way of doing this via the UI and query string? (I appreciate this could be done via api calls, but this isn't an option for my workflow).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @asos-danielc ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by EdGoldstein** [comment]
I would also really appreciate this feature addition.

It would also be very useful if there was an option to include cherry-picked commits into the release notes, and not just pull-requests.
